cmd/tink-worker/worker.go

```diff
@@ -22,6 +22,12 @@ const (
 	dataDir                  = "/worker"
 	maxFileSize              = "MAX_FILE_SIZE" // in bytes
 	defaultMaxFileSize int64 = 10485760        //10MB ~= 10485760Bytes
+
+	errGetWfContext       = "failed to get workflow context: %v"
+	errGetWfActions       = "failed to get for workflow: %s"
+	errReportActionStatus = "failed to report action status: %v"
```

Comment:
Mostly the same as above, though `errGetWfActions` is returned as an error here which I think just needs to be treated like a special case. Having log lines with the context values properly is necessary.